### ISSUE TITLE
Make GetResponse in query package return a defined interface

### ISSUE THREAD

**Issue by akhtariev** [dhiaayachi/go-newton-co#8]
More information about this issue can be found in the following [PR comment](https://github.com/dhiaayachi/go-newton-co/pull/7#discussion_r748961339)

**A/C**
- new interface is defined wrapping `map[string]interface{}`
- `GetAsString` and `GetAsFloat` implemented

**Comment by dhiaayachi** [comment]
After some thoughts I think the best way would be to make the user pass a response object that we would populate.

currently the user need to do something like this:

```go
        newton := newton.New("", "")
	q := &query.OpenOrders{}
	response, _ := newton.Do(q)
	resp := response.Body.(*query.OpenOrdersResponse)
	fmt.Println((*resp)[0].Price)
```
Which I think is not ideal, a better way to do this would be:

```go
        newton := newton.New("", "")
	q := &query.OpenOrders{}
        r := query.OpenOrdersResponse{}
	responseCode, _ := newton.Do(q,&r)
	fmt.Println(r[0].Price)
```

What you think about this @akhtariev ?

**Comment by akhtariev** [comment]
That looks good to me. Maybe we could even make it as part of a `response` package? So it would be:
```
r := response.OpenOrders{}
```

**Comment by dhiaayachi** [comment]
Yes great idea, I will work on this when I have some spare time.
